src/Badge/Badge.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import cx from '../utils/classnames';
+import { props, t } from '../utils';
+import FlexView from 'react-flexview';
+
+export type BadgeProps = {
+  /** the descriptive content of the badge */
+  label: React.ReactNode,
+  /** tells if the badge is active (for styling purposes) */
+  active?: boolean,
+  /** an optional class name to pass to top level element of the component */
+  className?: string,
+  /** an optional style object to pass to top level element of the component */
+  style?: React.CSSProperties
+};
+
+export const Props = {
```

Comment:
yes, many projects are still using this.
the idea is: the code will be refactored in TS but projects not in TS should be able to upgrade anyway